docs/src/components/landing/PricingTable.js

```diff
@@ -0,0 +1,451 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { alpha } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Fade from '@mui/material/Fade';
+import Paper from '@mui/material/Paper';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Divider from '@mui/material/Divider';
+import Typography from '@mui/material/Typography';
+import IconImage from 'docs/src/components/icon/IconImage';
+
+function PlanName({ planInfo, centered = false, disableDescription = false }) {
+  const { title, color, description } = planInfo;
+  return (
+    <React.Fragment>
+      {centered ? (
+        <Typography
+          variant="body2"
+          fontWeight="bold"
+          sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}
+        >
+          <IconImage name={`block-${color}`} sx={{ mr: 1 }} /> {title}
+        </Typography>
+      ) : (
+        <Typography
+          variant="body2"
+          fontWeight="bold"
+          sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}
+        >
+          {title} <IconImage name={`block-${color}`} />
+        </Typography>
+      )}
+      {!disableDescription ? (
+        <Typography variant="body2" color="text.secondary" sx={{ mt: 0.5, minHeight: { md: 63 } }}>
+          {description}
+        </Typography>
+      ) : null}
+    </React.Fragment>
+  );
+}
+
+PlanName.propTypes = {
+  centered: PropTypes.bool,
+  disableDescription: PropTypes.bool,
+  planInfo: PropTypes.shape({
+    color: PropTypes.string.isRequired,
+    description: PropTypes.string,
+    title: PropTypes.string.isRequired,
+  }),
+};
+
+const Cell = ({ highlighted = false, ...props }) => (
```

Comment:
not a big deal but just for consistency - the component above uses the `function` keyword while this one is an arrow function, we should probably always write components the same way